aea/cli/test.py

```diff
@@ -366,30 +434,20 @@ def test_package_by_path(
         packages_dir,
         skip_consistency_check=skip_consistency_check,
     )
-    with CoveragercFile(root_dir=cov_output) as covrc_file:
-        with cd(package_dir):
```

Comment:
We don't need to change the directory to the package since we're loading the required modules before and running the tests in the working directory also allows to merge the coverage data with previous runs using the `--append` command